pandora-common/src/assets/item.ts

```diff
@@ -1276,6 +1277,14 @@ export class ItemLock extends ItemBase<'lock'> {
 		}
 		return true;
 	}
+
+	private static _isEqualPassword(_asset: Asset<'lock'>, lhs?: string, rhs?: string): boolean {
+		if (lhs == null || rhs == null)
+			return lhs === rhs;
+
+		// all passwords are case insensitive for now
+		return lhs.localeCompare(rhs, undefined, { sensitivity: 'base' }) === 0;
```

Comment:
I would really like to avoid any `locale*` function in the common - whole principle of `locale*` functions is that they behave differently based on the machine/browser's locale, which we definitely don't want for checks like this.
I see two solutions:
- Instead of `undefined` use some specific locale (the second argument)
- Use a simple way of doing `.toLowerCase()` on both strings.